Makefile

```diff
@@ -32,7 +32,7 @@ SRC_DIRS       = $(shell sh -c "find $(TOP_SRC_DIRS) -name \\*.go \
                    -exec dirname {} \\; | sort | uniq")
 TEST_DIRS     ?= $(shell sh -c "find $(TOP_SRC_DIRS) -name \\*_test.go \
                    -exec dirname {} \\; | sort | uniq")
-VERSION       ?= $(shell git describe --always --abbrev=7 --dirty)
+VERSION       ?= $(shell git describe --tag --always --abbrev=7 --dirty)
```

Comment:
We upload our images from CI with just the commit sha in the [name](https://github.com/kubernetes-incubator/service-catalog/blob/master/contrib/jenkins/build.sh#L41), this change misaligns the naming with the ones that we [upload](https://quay.io/repository/kubernetes-service-catalog/controller-manager?tab=tags).

Probably nothing is wrong with this, just want to make a note here for discussion.